### PR TITLE
Fix for REST Server pstack api

### PR DESCRIPTION
**Pull request by venkat1m**
Fix for TRAFODION-1814 : REST Servers' pstack API does not return pstack information of process

https://issues.apache.org/jira/browse/TRAFODION-1814

### CODE REVIEW

File: core/rest/src/main/java/org/trafodion/rest/ServerResource.java [hunk 1]
```diff
@@ -256,11 +246,13 @@ private JSONArray pstack(String program) throws IOException {
 	            String line = scanner.nextLine();
 	            if(line.contains("pstack-ing")) {
 	                continue;
-	            } else if (line.contains("pstack")) {
-	                if(pstack == true) {
+				} else if (line.contains("pstack") || line.startsWith("--")) {
+					if (pstack == true && sb.length() > 0) {
```
Comment (DaveBirdsall) [inline core/rest/src/main/java/org/trafodion/rest/ServerResource.java]: This change looks good to me. I gather that all the changes with the imports are just clean-ups unrelated to this change?

**Comment by venkat1m** [comment]
Yes, the import cleanups were automatically done by eclipse and unrelated to this change.

**Comment by venkat1m** [comment]
The fix was validated by checking the JSON response to the following requests:
http://<host>:4200/v1/servers/pstack
http://<host>:4200/v1/servers/pstack/program/<pid>

**Comment by arvind-narain** [comment]
Looks good Venkat - thanks. Minor on white-spacing around 249/250, but that shouldn't hold up the merge.

**Comment by arvind-narain** [comment]
Merging in now.

**Pull request merged**